scripts/debug_course_publisher.py

```diff
@@ -7,33 +7,50 @@
 from geometry_msgs.msg import TransformStamped
 
 from mrover.msg import Course, Waypoint
+import mrover.srv
 
 
-def send_waypoint(frame_id: str, x: float, y: float, fid_id: int = -1) -> Waypoint:
+def add_waypoint(frame_id: str, x: float, y: float, fid_id: int = -1) -> Waypoint:
```

Comment:
`x` and `y` look unused in this function, so unless I am missing something, seems they should be removed.